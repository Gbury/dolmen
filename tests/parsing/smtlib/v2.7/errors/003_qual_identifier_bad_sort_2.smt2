(set-logic ALL)
(assert (as foo !))
