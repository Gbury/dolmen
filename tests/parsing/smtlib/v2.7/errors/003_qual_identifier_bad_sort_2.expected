File "tests/parsing/smtlib/v2.7/errors/003_qual_identifier_bad_sort_2.smt2", line 2, character 16-17:
2 | (assert (as foo !))
                    ^
Error while parsing a qualified identifier, read the reserved word '!',
      but expected a sort, i.e. either an identifier, or a sort constructor
      application; note that keywords (such as '_', 'as', ...) are none of
      these, and thus are not allowed here..
